### PR TITLE
Create `List` component

### PR DESCRIPTION
**Pull request by daoanhhuy26012001**
- [x] Create `ListBrand` & `ListProduct`
- [x] Consider styles for list
- [x] Change the location of the folder (common -> sections)

### CODE REVIEW

File: ReactJS/practice-one/src/components/Sections/ListBrand/index.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Styles
+import './listBrand.css';
+import { Button } from '../../Common/Button';
+import { VARIANTS } from '../../../constants/constant';
+
+export const List = ({ brands }) => {
```
Comment (phattruong-agilityio) [inline ReactJS/practice-one/src/components/Sections/ListBrand/index.jsx:6]: Check naming here

File: ReactJS/practice-one/src/components/Sections/ListProduct/index.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Styles
+import './listProduct.css';
+import { ProductCard } from '../../Common/ProductCard';
+
+export const listProduct = ({ products }) => {
```
Comment (phattruong-agilityio) [inline ReactJS/practice-one/src/components/Sections/ListProduct/index.jsx:5]: Check naming here

File: ReactJS/practice-one/src/components/Sections/ListProduct/index.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  return (
+    <div className="list-product">
+      {products.map((product, index) => (
+        <div key={index}>
```
Comment (phattruong-agilityio) [inline ReactJS/practice-one/src/components/Sections/ListProduct/index.jsx:9]: key is not a index. Please re-read https://legacy.reactjs.org/docs/lists-and-keys.html

File: ReactJS/practice-one/src/components/Sections/ListBrand/index.jsx (hunk 1 above)
Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/components/Sections/ListBrand/index.jsx:6]: Done

Comment (phattruong-agilityio) [reply ReactJS/practice-one/src/components/Sections/ListBrand/index.jsx:6]: Update the file name also.

File: ReactJS/practice-one/src/components/Sections/ListProduct/index.jsx (hunk 2 above)
Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/components/Sections/ListProduct/index.jsx:5]: Done

File: ReactJS/practice-one/src/components/Sections/ListBrand/index.jsx (hunk 1 above)
Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/components/Sections/ListBrand/index.jsx:6]: What? 
I think the CSS file names don't need to capitalize first letter

File: ReactJS/practice-one/src/components/Sections/ListProduct/index.jsx (hunk 3 above)
Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/components/Sections/ListProduct/index.jsx:9]: Done

**Pull request merged**